napi-inl.h

```diff
@@ -3522,6 +3522,32 @@ inline AsyncWorker::AsyncWorker(const Object& receiver,
   NAPI_THROW_IF_FAILED_VOID(_env, status);
 }
 
+inline AsyncWorker::AsyncWorker(napi_env env)
```

Comment:
These should be `Napi::Env env` as opposed to `napi_env env` for consistency with the rest of the API.  Same goes for other places env is being added.

Reply:
Thanks for the tip. I was a little confused because several places use `napi_env` but maybe those are older pieces of code. Changed now to `Napi::Env`